src/db/spanner/models.rs

```diff
@@ -1266,8 +1266,8 @@ impl SpannerDb {
                 } else {
                     "".to_string()
                 }
-            )
-            .to_string();
```

Comment:
these are all related to the same clippy error: [redundant clone](https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone) - guessing it's the rust version bump that triggered these, since none of this is new code.